week01/leetcode01.cpp

```diff
@@ -1 +1,19 @@
+// 1. Two Sum >> https://leetcode.com/problems/two-sum/
+// Time: O(N) where N is the size of the array
+// Space: O(N) where N is the size of the array
 
+class Solution {
+public:
+    vector<int> twoSum(vector<int>& nums, int target) {
+        unordered_map<int, int> hashTable;
```

Comment:
We all know it is a `hashTable`. Choose a meaningful name instead?
```suggestion
        unordered_map<int, int> seen;
```